### PR TITLE
feat(oui-icons): add guides icon

### PR DESCRIPTION
**Pull request by marie-j**
### Description of the Change

 Add guides icon

### CODE REVIEW

File: packages/oui-icons/svg/guides.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
+    <!-- Generator: Sketch 54 (76480) - https://sketchapp.com -->
+    <title>Icons/System/book</title>
+    <desc>Created with Sketch.</desc>
+    <g id="Icons/System/book" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <path d="M20,18.0001 L15.578,18.0001 L17.483,16.8571 C17.784,16.6771 17.969,16.3521 17.969,16.0001 L17.969,9.0001 L20,9.0001 L20,18.0001 Z M4,9.0001 L6.031,9.0001 L6.031,16.0001 C6.031,16.3501 6.213,16.6731 6.512,16.8541 L8.396,18.0001 L4,18.0001 L4,9.0001 Z M8.031,6.7781 L10.967,8.5621 L10.967,17.2221 L8.031,15.4371 L8.031,6.7781 Z M12.967,8.5661 L15.969,6.7661 L15.969,15.4341 L12.967,17.2341 L12.967,8.5661 Z M20,7.0001 L17.969,7.0001 L17.969,5.0001 C17.969,4.6401 17.775,4.3081 17.461,4.1301 C17.148,3.9511 16.763,3.9581 16.455,4.1431 L11.971,6.8321 L7.551,4.1461 C7.242,3.9591 6.856,3.9531 6.541,4.1281 C6.226,4.3061 6.031,4.6391 6.031,5.0001 L6.031,7.0001 L4,7.0001 C2.897,7.0001 2,7.8971 2,9.0001 L2,18.0001 C2,19.1031 2.897,20.0001 4,20.0001 L20,20.0001 C21.103,20.0001 22,19.1031 22,18.0001 L22,9.0001 C22,7.8971 21.103,7.0001 20,7.0001 L20,7.0001 Z" id="Color" fill="#2859C0"></path>
+    </g>
+</svg>
```
Comment (FredericEspiau) [inline packages/oui-icons/svg/guides.svg]: Line carriage

**Comment by AxelPeter** [comment]
It could be a nice add to update the icon used by the guide component here: https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/guide-menu.

For now, it's directly the SVG: https://github.com/ovh-ux/ovh-ui-angular/blob/master/packages/oui-guide-menu/src/guide-menu.html#L4.
We could use this icon instead.

File: packages/oui-icons/svg/guides.svg (hunk 1 above)
Comment (AxelPeter) [reply packages/oui-icons/svg/guides.svg]: Since it's the generated SVG file given by CX, I think we could ignore this :)

Comment (FredericEspiau) [reply packages/oui-icons/svg/guides.svg]: Hm I don't agree, we should have a tool that does it though :)

**Pull request merged**